packages/examples/common/ngComponentOutlet/ts/module.ts

```diff
@@ -33,22 +33,28 @@ export class Greeter {
 
 @Component({
   selector: 'complete-component',
-  template: `Complete: <ng-content></ng-content> <ng-content></ng-content>{{ greeter.suffix }}`
+  template: `{{ label }}: <ng-content></ng-content> <ng-content></ng-content>{{ greeter.suffix }}`
 })
 export class CompleteComponent {
+  @Input() label: string;
+
   constructor(public greeter: Greeter) {}
 }
 
 @Component({
   selector: 'ng-component-outlet-complete-example',
   template: `
     <ng-container *ngComponentOutlet="CompleteComponent;
+                                      inputs: myInputs;
                                       injector: myInjector;
                                       content: myContent"></ng-container>`
 })
 export class NgComponentOutletCompleteExample {
   // This field is necessary to expose CompleteComponent to the template.
   CompleteComponent = CompleteComponent;
+
+  myInputs = {label: 'Complete'};
```

Comment:
```suggestion
  myInputs = {'label': 'Complete'};
```

The name should be quoted or a compiler with property renaming might come by and break things since the mirror _won't_ have renamed properties for the "templateName"